lighthouse-core/config/default-config.js

```diff
@@ -457,6 +458,8 @@ const defaultConfig = {
         {id: 'password-inputs-can-be-pasted-into', weight: 1},
         {id: 'errors-in-console', weight: 1},
         {id: 'image-aspect-ratio', weight: 1},
+        // Manual audits
+        {id: 'polyfills', weight: 0},
```

Comment:
IMO this belongs in performance as a diagnostic and can be upgraded to an opportunity once we have the size data :)